src/extension.ts

```diff
@@ -5,6 +5,7 @@ import { Socket } from 'net';
 
 import { workspace, ExtensionContext, window } from 'vscode';
 import { LanguageClient, LanguageClientOptions, ServerOptions, StreamInfo } from 'vscode-languageclient/node';
+import * as vscode from 'vscode';

```

Comment:
There's already a vscode import. Could you add the bits you need there directly?